scripts/FlowByActivity_Crosswalks/write_Crosswalk_USDA_CoA_Livestock.py

```diff
@@ -33,8 +33,8 @@ def assign_naics(df):
     # dual-purpose cattle ranching and farming: 11213
     df.loc[df['Activity'] == 'CATTLE, (EXCL COWS)', 'Sector'] = '112130A'
     df.loc[df['Activity'] == 'CATTLE, COWS', 'Sector'] = '112130B'
-    df.loc[df['Activity'] == 'CATTLE, COWS, BEEF', 'Sector'] = '112130B1'
-    df.loc[df['Activity'] == 'CATTLE, COWS, MILK', 'Sector'] = '112130B2'
+    # df.loc[df['Activity'] == 'CATTLE, COWS, BEEF', 'Sector'] = '112130B1'
+    # df.loc[df['Activity'] == 'CATTLE, COWS, MILK', 'Sector'] = '112130B2'
```

Comment:
are these supposed to be removed?

Reply:
> I wonder if we can re-think this file a bit. Right now because each section is numbered making a small change results in lots of line updates so its hard to track. I suspect a lot of these change in `fbs_yaml_refac`
> 
> Bullets might be better as in most cases the order doesn't matter. Maybe we could lead with the required fields followed by the optional fields and/or use a table.

I like this idea. I see I updated this file prematurely with the revised parameters in the fbs_refac_branch